Mathlib/MeasureTheory/Decomposition/RadonNikodym.lean

```diff
@@ -302,6 +302,9 @@ lemma setLIntegral_rnDeriv_le (s : Set α) :
     ∫⁻ x in s, μ.rnDeriv ν x ∂ν ≤ μ s :=
   (withDensity_apply_le _ _).trans (Measure.le_iff'.1 (withDensity_rnDeriv_le μ ν) s)
 
+lemma lintegral_rnDeriv_le : ∫⁻ x, μ.rnDeriv ν x ∂ν ≤ μ .univ :=
```

Comment:
What is this `.univ` stuff? Is that `Set.univ` in disguise? How does Lean guess that it should add in `Set`?

Ha, it's probably because Lean knows that it expects a Set, so it uses the fancy dot notation there.

I'd maybe switch to `Set.univ` here and in the proof, to avoid making people wonder what is going on. This is completely optional, though.

Reply:
Yes Lean knows that the argument should be a set, so it inserts Set. That's "anonymous dot notation".